src/launcher/util/appCompatibilityWarning.tsx

```diff
@@ -0,0 +1,292 @@
+                appVersion: app.currentVersion,
+                minimalAppVersion: 'none',
+            }
+        );
+    }
+
+    const appIsRecentEnough =
+        minimalRequiredAppVersions[app.name] == null ||
+        semver.gte(app.currentVersion, minSupportedVersion);
+
+    const fittingVersionExists =
+        minimalRequiredAppVersions[app.name] != null &&
+        isDownloadable(app) &&
+        !isWithdrawn(app) &&
+        app.latestVersion != null &&
+        semver.gte(app.latestVersion, minSupportedVersion);
+
+    return appIsRecentEnough
+        ? undecided
+        : incompatible(
+              'Version problem',
+              'This version of nRF Connect for Desktop does not support ' +
+                  `version ${app.currentVersion} of this app. You ` +
+                  `need at least version ` +
+                  `${minimalRequiredAppVersions[app.name]}.`,
+              `This version of nRF Connect for Desktop does not support ` +
+                  `version ${app.currentVersion} of this app ` +
+                  `"${app.displayName}". You need at least version ` +
+                  `${minimalRequiredAppVersions[app.name]}.${
+                      fittingVersionExists
+                          ? ' Download the latest available version of this app.'
+                          : ''
+                  } Running the currently installed version will not work.`,
+              {
+                  warningKind: WarningKind.UNSUPPORTED_APP,
+                  app: app.name,
+                  appVersion: app.currentVersion,
+                  minimalAppVersion: minimalRequiredAppVersions[app.name],
+              }
+          );
+};
+
+const nrfutilDeviceToJLink = (device: string) => {
+    // According to https://docs.nordicsemi.com/bundle/nrfutil/page/guides/installing.html#prerequisites
+    if (semver.lt(device, '2.0.0')) {
+        return 'V7.80c';
+    }
+
+    if (semver.lt(device, '2.1.0')) {
+        return 'V7.88j';
+    }
+
+    if (semver.lt(device, '2.5.4')) {
+        return 'V7.94e';
+    }
+
+    return 'V7.94i';
+};
+
+const checkJLinkRequirements: AppCompatibilityChecker = async (
+    app: LaunchableApp
+) => {
+    if (!isDownloadable(app)) {
+        return undecided;
+    }
+
+    const deviceVersion =
+        app.versions?.[app.currentVersion]?.nrfutilModules?.device?.at(0);
+
+    if (!deviceVersion) {
+        return undecided;
+    }
+
+    const userDir = getUserDataDir();
+    const sandbox = await getSandbox(userDir, 'device', deviceVersion);
+    const moduleVersion = await sandbox.getModuleVersion();
+    const jlinkVersionDependency = resolveModuleVersion(
+        'JlinkARM',
+        moduleVersion.dependencies
+    );
+
+    if (!jlinkVersionDependency) {
+        const requiredVersion = nrfutilDeviceToJLink(deviceVersion);
+
+        return incompatible(
+            'Missing dependency',
+            `Required SEGGER J-Link not found, expected version ${requiredVersion}`,
```

Comment:
```suggestion
            `Required SEGGER J-Link not found: expected version ${requiredVersion}`,
```

Reply:
Above you suggested a semicolon in the same string. 🤔 I will go for the colon here and there. I you prefer the semicolon I can change it.